### PR TITLE
doc/md: add redshift

### PR DESCRIPTION
**Pull request by datdao**
Updated for Redshiftdata Urls:
Screenshot 2024-05-01 at 12 25 44

Updated for guide

https://github.com/ariga/atlas/assets/16095902/379a4bf1-7378-43cd-ac75-03f6a6cc5944

### CODE REVIEW

File: doc/md/atlas-schema/hcl-types.md [hunk 1]
```diff
@@ -1635,3 +1635,219 @@ table "t" {
   }
 }
 ```
+
+## Redshift
+
+### Boolean
+The `boolean` and `bool` type allows creating standard SQL boolean columns.
```
Comment (hilakashai) [inline doc/md/atlas-schema/hcl-types.md:1642]:
```suggestion
The `boolean` and `bool` types allow creating standard SQL boolean columns.
```

File: doc/md/atlas-schema/hcl-types.md [hunk 2]
```diff
@@ -1635,3 +1635,219 @@ table "t" {
...
+  }
+  column "c2" {
+    # Alias to boolean.
+    type = bool
+  }
+}
+```
+
+### Binary
+The `binary_varying`, `varbinary` and `varbyte` type allows creating binary string columns.
```
Comment (hilakashai) [inline doc/md/atlas-schema/hcl-types.md:1658]:
```suggestion
The `binary_varying`, `varbinary` and `varbyte` types allow creating binary string columns.
```

File: doc/md/atlas-schema/hcl.mdx [hunk 3]
```diff
@@ -230,6 +241,63 @@ table "b" "users" {
 }
 ```
 
+### Distribution
+
+The `distribution` attribute allows for specifying the distribution method of the table.
+
+:::note
+Distribution are currently supported only by the Redshift driver. Support for the remaining drivers will be added in future versions.
```
Comment (hilakashai) [inline doc/md/atlas-schema/hcl.mdx:249]:
```suggestion
Distribution is currently supported only by the Redshift driver. Support for the remaining drivers will be added in future versions.
```

File: doc/md/atlas-schema/hcl.mdx [hunk 4]
```diff
@@ -230,6 +241,63 @@ table "b" "users" {
...
+
+</TabItem>
+</Tabs>
+
+### Sorting
+
+The `sort` attribute allows for specifying the sorting method of the table.
+
+:::note
+Sorting are currently supported only by the Redshift driver. Support for the remaining drivers will be added in future versions.
```
Comment (hilakashai) [inline doc/md/atlas-schema/hcl.mdx:277]:
```suggestion
Sorting is currently supported only by the Redshift driver. Support for the remaining drivers will be added in future versions.
```

File: doc/md/concepts/dev.mdx [hunk 5]
```diff
@@ -161,6 +161,21 @@ docker+clickhouse://user/image:tag
 docker+clickhouse://_/clickhousedb:latest
 ```
 
+</TabItem>
+<TabItem value="redshift" label="Redshift">
+
+Redshift is a managed service and does not support running in a Docker container. To user dev-database with Redshift,
```
Comment (hilakashai) [inline doc/md/concepts/dev.mdx:167]:
```suggestion
Redshift is a managed service and does not support running in a Docker container. To use a dev-database with Redshift,
```

File: doc/md/concepts/dev.mdx [hunk 6]
```diff
@@ -161,6 +161,21 @@ docker+clickhouse://user/image:tag
...
+We recommend using a real Redshift cluster with separate database
```
Comment (hilakashai) [inline doc/md/concepts/dev.mdx:168]:
```suggestion
we recommend using a real Redshift cluster with separate database.
```

File: doc/md/concepts/url.mdx [hunk 7]
```diff
@@ -146,6 +147,19 @@ Connecting to a specific ClickHouse schema (database) with a username and passwo
 clickhouse://user:pass@localhost:9000/schema
 ```
 
+</TabItem>
+<TabItem value="redshift">
+
+Connecting to a specific Redshift cluster with schema named `public`:
```
Comment (hilakashai) [inline doc/md/concepts/url.mdx:153]:
```suggestion
Connecting to a specific Redshift cluster with a schema named `public`:
```

File: doc/md/concepts/url.mdx [hunk 8]
```diff
@@ -146,6 +147,19 @@ Connecting to a specific ClickHouse schema (database) with a username and passwo
...
+```shell
+redshift://user:pass@redshift-cluster:5439/database?search_path=public
+```
+
+Connecting to a specific Redshift cluster with schema named `public` with SSL disabled:
```
Comment (hilakashai) [inline doc/md/concepts/url.mdx:158]:
```suggestion
Connecting to a specific Redshift cluster with a schema named `public` with SSL disabled:
```

File: doc/md/declarative/diff.mdx [hunk 9]
```diff
@@ -201,6 +210,18 @@ atlas schema diff \
   --to "clickhouse://user:pass@remote:9000/example"
 ```
 
+</TabItem>
+<TabItem label="Redshift" value="redshift">
+
+Compare two Redshift clusters with schema named `public` under the
```
Comment (hilakashai) [inline doc/md/declarative/diff.mdx:216]:
```suggestion
Compare two Redshift clusters with a schema named `public` under the
```

**Review by hilakashai — COMMENTED** [review]
🚀 🚀

File: doc/md/versioned/diff.mdx [hunk 10]
```diff
@@ -807,6 +877,16 @@ CREATE TABLE `users` (
...
 ```
 
+</TabItem>
+<TabItem value="redshift" label="Redshift">
+
+```sql title="schema.sql" {3}
+CREATE TABLE `users` (
+  `id` int NOT NULL,
+  `name` varchar(255) NOT NULL
+)
```
Comment (a8m) [inline doc/md/versioned/diff.mdx:887]: This should be quoted with `"`, no?

**Review by rotemtam — APPROVED** [review]

**Pull request merged**